Mathlib/Init.lean

```diff
@@ -2,6 +2,9 @@ import Mathlib.Tactic.Linter.DocPrime
 import Mathlib.Tactic.Linter.GlobalAttributeIn
 import Mathlib.Tactic.Linter.HashCommandLinter
 import Mathlib.Tactic.Linter.Header
+-- This linter is disabled by default: we import it here so downstream projects can enable it
+-- project-wide without having to wonder if that linter was imported.
+import Mathlib.Tactic.Linter.FlexibleLinter
```

Comment:
If we do this, shouldn't we import `Mathlib/Tactic/Linter`? (and remove the comment from there)

Reply:
I have updated the comments and PR description to clarify this: in short, there are linters which I don't see a need to import in Mathlib.Init. (Whether that entire file should be removed and inlined in to Mathlib.Init is a different question: I'd rather have that conversation in a follow-up PR.)